test/arr.js

```
@@ -6,33 +6,33 @@ const original = new Arr(1, 2, 3)
-    await t.test('Test if the mappen result is a instance of Arr', () => {
-        assert.strictEqual(mapped instanceof Arr, false)
-    })
+  await t.test('Test if the mappen result is a instance of Arr', () => {
+    assert.strictEqual(mapped instanceof Arr, false)
+  })
 
-    await t.test('Test if the mappen result is a instance of Array', () => {
-        assert.strictEqual(mapped instanceof Array, true)
-    })
+  await t.test('Test if the mappen result is a instance of Array', () => {
+    assert.strictEqual(mapped instanceof Array, true)
```

Biome (1.9.4)

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)